install/helm/agones/templates/crds/_gameserverspecvalidation.yaml

```diff
@@ -92,6 +92,17 @@ properties:
               type: integer
               minimum: 1
               maximum: 65535
+      sdk-server-loglevel:
+        type: string
+        enum:
```

Comment:
I don't think we should expose so many log levels. This seems to be externalizing the fact that we are using logrus and would make it nearly impossible to change our logging library in the future. I feel like there are really three categories that people want:

1. Default (what we have today, which in basically the info level)
2. Errors and above
3. Debugging - everything

What if we abstracted the log levels from logrus and just offered something smaller like those three? Alternatively, we could give people a way to specifically disable the ping logs since those are the ones that cause the most noise in the log file.

Reply:
Yes, less level would be more concise. I will do the change cause that would actually give more space for the changes in the future.